doc/documentation/format_versions/biom-2.1.rst

```diff
@@ -60,7 +60,7 @@ Under the ``observation/group-metadata`` and ``sample/group-metadata`` groups, t
 Example biom files
 ==================
 
-Below is an example of a rich biom file.
+Below is an example of a rich biom file before binary compression into HDF5. Example binary files are located in the `github repository <https://github.com/biocore/biom-format/tree/master/examples>`_.
```

Comment:
How's about:

"Below is an example of a rich BIOM file before as represented by the HDF5 Data Description Language (DDL). Examples of actual BIOM 2.1 files can be found in the `github repository <https://github.com/biocore/biom-format/tree/master/examples>`_"